install/page-index/page-index.js

```diff
@@ -104,6 +104,15 @@ module.exports = [
         excerpt:
           "Install self-hosted TimescaleDB with a pre-built Docker container",
       },
+      {
+        title: "Kubernetes",
+        href: "installation-kubernetes",
+        title: "TimescaleDB on Kubernetes",
```

Comment:
```suggestion
        title: "Kubernetes",
```

For parallelism with the other section titles:
![image](https://user-images.githubusercontent.com/3914967/157370924-839e967a-4042-426a-ac6f-559595825cad.png)